project/depend.scala

```diff
@@ -5,14 +5,15 @@ object depend {
   val scalaz = Seq(  "org.scalaz" %% "scalaz-core"   % "7.1.0"
                    , "org.scalaz" %% "scalaz-effect" % "7.1.0")
 
-  val mundaneVersion = "1.2.1-20150318063632-5e03317"
+  val mundaneVersion = "1.2.1-20151104055659-410bb41"
   val mundane = Seq(
       "com.ambiata" %% "mundane-io"
     , "com.ambiata" %% "mundane-control").map(_ % mundaneVersion) ++ Seq(
       "com.ambiata" %% "mundane-io" % mundaneVersion % "test->test")
 
   val mundaneTesting = Seq(
-      "com.ambiata" %% "mundane-testing"   % mundaneVersion)
+      "com.ambiata" %% "mundane-testing"   % mundaneVersion % "test->test",
+      "com.ambiata" %% "mundane-testing"   % mundaneVersion % "compile->test")
```

Comment:
Because for now `saws-testing` also has test classes in `src/main`. Once the `mundane` PR is merged I will fix that as well.